.travis.yml

```diff
@@ -5,6 +5,10 @@ services:
   - docker
 language: go
 go_import_path: go.uber.org/yarpc
+branches:
+  only:
+    - master
+    - dev
```

Comment:
I think this might have had the unexpected consequence of only running tests on PRs which are based off of dev or master, which means that all stacked PRs won't run tests...  this is something I noticed after I returned from my break, has there been any other changes to travis since I left for vacation?